randovania/games/cave_story/logic_database/Ruined Egg Corridor.txt

```diff
@@ -277,10 +283,14 @@ Extra - starting_script: <TRA0053:0092:0165:0004
       Trivial
   > Room Top
       Any of the following:
+          # https://youtu.be/-hZwtVL18l0
```

Comment:
I would prefer if this video were not here, but instead below as `Damageless: https://youtu.be/-hZwtVL18l0?t=36`

Reply:
the vid was moved on purpose as the other strats are just more lenient variations of it with room for error

Reply:
I'll be honest I actually wanted to do it that way but I didn't want to have to redo all the trick reqs to add the extra ANY group, and yeah I don't think it looks that much better anyway